### ISSUE TITLE
bug: non-Next versions of Optimize documentation link to incorrect version of main docs in header

### ISSUE THREAD

**Issue by pepopowitz** [camunda/camunda-docs#1333]
Top-nav links (Guides, Components, etc) on all versions of Optimize docs link to the Next version of main documentation. This is correct for the Next version of Optimize docs, but incorrect for all other versions.

## Reproduction

Visit https://docs.camunda.io/optimize/apis-clients/optimize-api/optimize-api-authorization/

Expected result: top-nav links (Guides, Components, etc) go to **current** version of main documentation 
Actual result: top-nav links go to **Next** version of main documentation

## Implementation thoughts

I think this might be fixable in src/theme/NavbarItem/index.js, or another component inside NavbarItem. Like we did for the docsVersionDropdown in that file, we might need to alter some props we pass through to the link components when viewing optimize pages. 

cc @akeller for visibility

**Comment by pepopowitz** [comment]
@akeller when we last discussed this issue in person, we said we wanted to see proof that people were running into this issue before we decided to spend time on it. Have you heard from anyone who has unwittingly discovered this problem?

**Comment by akeller** [comment]
@pepopowitz thanks for checking in. I haven't heard anything.

**Comment by pepopowitz** [comment]
Cool, thanks. I'm inclined to de-prioritize this a bit, do you disagree?

**Comment by akeller** [comment]
Yep! You can even move it from "Ready" into "Backlog" to reinforce that.

**Comment by akeller** [comment]
Internal feedback - https://camunda.slack.com/archives/C026U8GBNSW/p1697469310379909

**Comment by pepopowitz** [comment]
Note: it's not always the Next version that those URLs go to. It's based on which version of the main docs you'd most recently browsed, because Docusaurus is trying to be smart about which version you want to see.

**Comment by pepopowitz** [comment]
More feedback: https://camunda.slack.com/archives/C026U8GBNSW/p1700684071839529
